### PR TITLE
merge me when Groebner@2.8 is out

### PR DESCRIPTION
**Pull request by sumiya11**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -25,7 +25,7 @@ TestSetExtensions = "98d24dd4-01ad-11ea-1b02-c9a08f80db04"
 AbstractAlgebra = "0.13, 0.18, 0.19, 0.20, 0.21, 0.22, 0.23, 0.24, 0.25, 0.26, 0.27"
 BenchmarkTools = "1"
 DataStructures = "0.18"
-Groebner = "0.2.4"
+Groebner = "0.2.8"
```
Comment (iliailmer) [inline Project.toml:28]: Should we make this just "0.2" to simplify managing versions?

Comment (sumiya11) [reply Project.toml:28]: Is `0.2` equivalent to `[0.2.0-0.3.0)` ?

Comment (sumiya11) [reply Project.toml:28]: I think `0.2.8` (`[0.2.8, 0.3.0)`) is good enough, since versions `0.2.7` and before fail SI.jl tests

Comment (iliailmer) [reply Project.toml:28]: Yes, I believe so.

Comment (sumiya11) [reply Project.toml:28]: sounds good to me

**Pull request merged**